docs/index.md

```diff
@@ -24,6 +24,11 @@ Subsequent examples will assume your project has a directory structure similar t
     └── index.php
 ```
 
+Instead of installing Equip via Composer, you can get going a little faster by using the [sample project](https://github.com/equip/project) that shows the best way to use this framework.
```

Comment:
I like that language change. I had originally stolen this line from the project README.